packages/core/src/services/ai/estimateCost/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Providers } from '@latitude-data/core'
```

Comment:
🙏 Let's try not to use package name inside the same package

Reply:
it all depends on the order of path alias declarations in the tsconfig file. If "@latitude-data/core" path alias is ever declared before "core/*", then the lsp will suggest the first match it finds which is in "@latitude-data/core", changing the path alias declaration order fixes the issue but it's very brittle.